package/opt/hassbian/suites/cloud9.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+function cloud9-show-short-info {
+  printf "Cloud9 install script for Hassbian.\\n"
+}
+
+function cloud9-show-long-info {
+  printf "Installs Cloud9 IDE onto this system.\\n"
+  printf "Cloud9 IDE is an online integrated development environment. It supports hundreds of programming languages.\\n"
```

Comment:
Rephrase according to the description changes.

Reply:
Changed in next commit.